.zenodo.json

```diff
@@ -0,0 +1,18 @@
+{
+    "creators": [
+        {
+            "name": "OMC-Team",
+            "affiliation": "CERN"
+        },
+    ],
+    "description": "Useful tools in particular for accelerator physicists at CERN.",
```

Comment:
We may want a description more fitting of a citation in this file.

Reply:
I'd argue that we should have there the same description as in the repo. If we change one, we change both